tools/tracer/tools/configure/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required( VERSION 2.6.2 FATAL_ERROR )
+project(mfx-tracer-config)
+
+set(headers "strfuncs.h" "config_manager.h")
+set(sources "config_manager.cpp" "main.cpp")
+
+set(no_warnings "-Wno-unknown-pragmas -Wno-unused")

```

Comment:
if you use make_executable, why this all is not done automatically?

Reply:
fixed